SourceX/miniwin/misc.cpp

```diff
@@ -120,7 +120,11 @@ WINBOOL DeleteFileA(LPCSTR lpFileName)
 
 bool SpawnWindow(LPCSTR lpWindowName, int nWidth, int nHeight)
 {
+#if defined(__3DS__)
+	if (SDL_Init(SDL_INIT_TIMER | SDL_INIT_VIDEO) <= -1) {

```

Comment:
Do you know what specific subsystem is causing issues?
Possible options are:
```
SDL_INIT_AUDIO
SDL_INIT_JOYSTICK
SDL_INIT_GAMECONTROLLER
SDL_INIT_EVENTS
SDL_INIT_NOPARACHUTE
```

Reply:
Let me get back on this.
But from my limited experience, i thought SDL_INIT_GAMECONTROLLER wasn't available at all in sdl 1.2.
And SDL_INIT_JOYSTICK should be supported, and probably also be enabled.

Reply:
True, I took the list from SDL2's documentation.

Reply:
SDL_INIT_CDROM seems to be the issue.

The others work without issue.
SDL_INIT_TIMER
SDL_INIT_AUDIO
SDL_INIT_VIDEO
SDL_INIT_JOYSTICK

These two are unaffected by SDL_INIT_EVERYTHING
SDL_INIT_NOPARACHUTE
SDL_INIT_EVENTTHREAD